lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
Correct. But the only way the data of an issue has a bugzilla_id is if I actually injected it. The query to redmine never pulls back issues with a bugzilla_id.

```redmine_urls = bugs['result']['bugs'].collect { |bug| {:url => bug['url'], :bugzilla_id => bug['id']} }```

I associated the bugzilla_id with the redmine url myself.